cilium-cli/connectivity/builder/bgp_control_plane.go

```diff
@@ -14,24 +14,13 @@ type bgpControlPlane struct{}
 
 func (t bgpControlPlane) build(ct *check.ConnectivityTest, _ map[string]string) {
 	// prefix the test name with `seq-` to run it sequentially
-	newTest("seq-bgp-control-plane-v1", ct).
+	newTest("seq-bgp-control-plane", ct).
 		WithCondition(func() bool {
 			return versioncheck.MustCompile(">=1.16.0")(ct.CiliumVersion) && ct.Params().IncludeUnsafeTests
 		}).
 		WithFeatureRequirements(
 			features.RequireEnabled(features.BGPControlPlane),
 			features.RequireEnabled(features.NodeWithoutCilium),
 		).
-		WithScenarios(tests.BGPAdvertisements(1))
-
-	// prefix the test name with `seq-` to run it sequentially
-	newTest("seq-bgp-control-plane-v2", ct).
-		WithCondition(func() bool {
-			return versioncheck.MustCompile(">=1.16.0")(ct.CiliumVersion) && ct.Params().IncludeUnsafeTests
-		}).
-		WithFeatureRequirements(
-			features.RequireEnabled(features.BGPControlPlane),
-			features.RequireEnabled(features.NodeWithoutCilium),
-		).
-		WithScenarios(tests.BGPAdvertisements(2))
+		WithScenarios(tests.BGPAdvertisements())
```

Comment:
> Since BGPv1 implementation has been removed, we should not
test it, nor collect BGPv1 resources in the sysdump.

But do we still want to test it on v1.16 / v1.17 / v1.18 ? The CLI that we're building from `main` ends up being used across all those versions.